pkg/proxy/nftables/proxier.go

```diff
@@ -576,7 +581,6 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 // It returns true if an error was encountered. Errors are logged.
 func CleanupLeftovers() bool {
 	var encounteredError bool
-
```

Comment:
looks like you started thinking about this and then didn't finish?

Reply:
Yeah, I don't know howto handle it (yet). You must have some way to say _which_ tables to delete, and prevent disasters when a user wants to cleanup a specific instance, but cleanup the default one by mistake

Reply:
I will leave this as-is for now. My idea is to _require_ `--nftables-table-name` to be specified when cleanup for proxy-mode=nftables is used